composer.json

```diff
@@ -30,5 +30,10 @@
       "Peridot\\": "src/"
     }
   },
+  "autoload-dev": {
```

Comment:
It will be ignored, yes. That's by design, as the polyfill is only needed for test coverage under PHP 5. I have tested this branch in another project under both PHP 5 and 7, and it works fine.